netaddr.go

```diff
@@ -666,6 +666,27 @@ func (ip IP) IsGlobalUnicast() bool {
 		!ip.IsLinkLocalUnicast()
 }
 
+// IsPrivate reports whether ip is a private address, according to RFC 1918
+// (IPv4 addresses) and RFC 4193 (IPv6 addresses).
```

Comment:
// ... That is, it reports whether ip is in 10.0.0.0/8, 192.168.0.0/16, .... or fc00::/7. This is the same
// as the standard library's net.IP.IsPrivate.